SpooderShooter/Assets/Scripts/Players/Movement.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+
+public class Movement : MonoBehaviour {
+    private float speed = 1;
```

Comment:
More of a fun fact than anything, put if you change this to [SerializeField] private float speed = 1; you can edit the value in the inspector.

Reply:
Super cool, just looked it up. Adding now!